assets/js/plugin-check-admin.js

```diff
@@ -172,14 +172,41 @@
 	 * @param {Object} data The response data.
 	 */
 	async function runChecks( data ) {
+		let isSuccessMessage = true;
+		let messageClass = '';
+		let messageText = '';
```

Comment:
@mukeshpanchal27 if we're defaulting to a success message would it make sense to include the success values for the class and text by default too? That way we don't need the else statement later.

```suggestion
		let isSuccessMessage = true;
		let messageClass = 'notice-success';
		let messageText = pluginCheck.successMessage;
```